src/components/FileRepo/FileRepo.js

```diff
@@ -50,7 +53,33 @@ const customTableTypes = {
   ),
 };
 
-const FileRepo = compose(injectState, withTheme, withApi)(
+const generateHeaderContent = (props, sqon) => {
+  props.disabled = false; //props.selectedTableRows == 0; @NOTE Keep functionality for next iteration of the ticket
+  props.sqon = sqon;
+  const cavaticaButton = (<Tooltip
+      position="top"
+      hideTitle
+      html={<Row p={'10px'}>{props.disabled ? 'Please select files in the table for this action.' : 'Cavatica is a cloud processing platform where files can be linked (not duplicated) and used immediately.'}</Row>}
+    >
+      <CavaticCopyButton {...props} />
+    </Tooltip>);
+
+  let downloadButton = (<DownloadButton {...props} />);
```

Comment:
Non-critical, but this seems simple enough for a `const`, `let` tends to create anxiety over potential mutation so we try to avoid it when possible :)